docs/guides/creatingadls.md

```diff
@@ -0,0 +1,140 @@
+# Creating a Bridge With a Dead Letter Sink (DLS)
+
+## What is a Dead Letter Sink?
+
+Knative Eventing provides various configuration parameters to control the delivery of events in case of failure. For instance, you can decide to retry sending events that failed to be consumed, and if this didn't work you can decide to forward those events to a dead letter sink.
```

Comment:
What do you think about replacing "Knative Eventing" with "TriggerMesh" here? 

The idea is that although this is provided by Knative under the hood, at this level we can hide that complexity from the user and just present it as a TriggerMesh capability.

Reply:
updated